src/app/layouts/navigation/drafts/Drafts.js

```diff
@@ -97,27 +98,31 @@ const Notification = ({ code: NOTIFICATION_GROUP, textColor }) => {
             )}
           </VStack>
         </MenuButton>
-
         <MenuList>
-          {!!isTenant
-            ? notificationWholeData.map(notification => (
+          {isNullOrUndefinedOrEmpty(notificationWholeData) &&
```

Comment:
all that is required is for you to check the notification count and only show the menulist if the notification count is equal or grater than 1.

ie: 
` {notificationCount >= 1 && (
          <MenuList>
            {!!isTenant
              ? notificationWholeData.map(notification => (
                  <NotificationHandler
                    key={notification?.name}
                    parentCode={NOTIFICATION_GROUP}
                    code={notification?.question?.code || ''}
                    sourceCode={sourceCode}
                    targetCode={targetCode}
                  />
                ))
              : notification
                  .reverse()
                  .map(notification => (
                    <NotificationHandler
                      key={notification}
                      parentCode={NOTIFICATION_GROUP}
                      code={notification}
                    />
                  ))}
          </MenuList>
        )}`

Reply:
I considered hiding the list outright, but the list not appearing when the button is clicked might be confusing for the user as it provides no feedback. Will implement the notificationCount check, was not aware of that